<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slicer</class>
 <widget class="QWidget" name="Slicer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>358</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Video File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileNameEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="openFileButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="inputLength">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="inputOffset">
        <property name="minimum">
         <number>-2147483647</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Input Template</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="templateInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="convertButton">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textOuput"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
